### PR TITLE
[BLD-1024] Build script added

### PR DESCRIPTION
**Pull request by EvgeniyPatlan**

### CODE REVIEW

File: percona-packaging/psmdb_builder.sh [hunk 1]
```diff
@@ -0,0 +1,784 @@
...
+Usage: $0 [OPTIONS]
+    The following options may be given :
+        --builddir=DIR      Absolute path to the dir where all actions will be performed
+        --get_sources       Source will be downloaded from github
+        --build_src_rpm     If it is 1 src rpm will be built
+        --build_source_deb  If it is 1 source deb package will be built
+        --build_rpm         If it is 1 rpm will be built
+        --build_deb         If it is 1 deb will be built
+        --build_tarball     If it is 1 tarball will be built
+        --install_deps      Install build dependencies(root previlages are required)
```
Comment (tplavcic) [inline percona-packaging/psmdb_builder.sh]: previlages -> privileges

File: percona-packaging/psmdb_builder.sh [hunk 2]
```diff
@@ -0,0 +1,784 @@
...
+    if test "$1" = PICK-ARGS-FROM-ARGV
+    then
+        pick_args=1
+        shift
+    fi
+  
+    for arg do
+        val=$(echo "$arg" | sed -e 's;^--[^=]*=;;')
+        case "$arg" in
+            # these get passed explicitly to mysqld
```
Comment (tplavcic) [inline percona-packaging/psmdb_builder.sh]: mysqld? maybe the whole comment should be removed

File: percona-packaging/psmdb_builder.sh (hunk 1 above)
Comment (tplavcic) [reply percona-packaging/psmdb_builder.sh]: why only 1 rpm/deb will be built?

Comment (EvgeniyPatlan) [reply percona-packaging/psmdb_builder.sh]: fixed

File: percona-packaging/psmdb_builder.sh (hunk 2 above)
Comment (EvgeniyPatlan) [reply percona-packaging/psmdb_builder.sh]: fixed

File: percona-packaging/scripts/psmdb_builder.sh [hunk 3]
```diff
@@ -0,0 +1,783 @@
...
+        --install_deps      Install build dependencies(root privilages are required)
+        --branch            Branch for build
+        --repo              Repo for build
+        --psm_ver           PSM_VER(mandatory)
+        --psm_release       PSM_RELEASE(mandatory)
+        --mongo_tools_tag   MONGO_TOOLS_TAG(mandatory)
+        --ftindex_tag       FTINDEX_TAG(mandatory)
+        --jemalloc_tag      JEMALLOC_TAG(mandatory)
+        --rocksdb_tag       ROCKSDB_TAG(mandatory)
+        --tokubackup_branch TOKUBACKUP_BRANCH(mandatory)
```
Comment (igorsol) [inline percona-packaging/scripts/psmdb_builder.sh]: Why branch is used for tokubackup while tags are used for other components?

File: percona-packaging/scripts/psmdb_builder.sh [hunk 4]
```diff
@@ -0,0 +1,783 @@
...
+        --debug             build debug tarball
+        
+        --help) usage ;;
+Example $0 --builddir=/tmp/PSMDB --get_sources=1 --build_src_rpm=1 --build_rpm=1
```
Comment (igorsol) [inline percona-packaging/scripts/psmdb_builder.sh]: Why example does not contain all those mandatory arguments like ftindex_tag, rocksdb_tag etc

File: percona-packaging/scripts/psmdb_builder.sh [hunk 5]
```diff
@@ -0,0 +1,783 @@
...
+        --build_src_rpm     If it is set - src rpm will be built
+        --build_source_deb  If it is set - source deb package will be built
+        --build_rpm         If it is set - rpm will be built
+        --build_deb         If it is set - deb will be built
+        --build_tarball     If it is set - tarball will be built
+        --install_deps      Install build dependencies(root privilages are required)
+        --branch            Branch for build
+        --repo              Repo for build
+        --psm_ver           PSM_VER(mandatory)
+        --psm_release       PSM_RELEASE(mandatory)
```
Comment (igorsol) [inline percona-packaging/scripts/psmdb_builder.sh]: It would be good to have some examples for --psm_ver and --psm_release

Comment (EvgeniyPatlan) [reply percona-packaging/scripts/psmdb_builder.sh]: I have added the README which includes all possible way to run this script to get all needed staff

**Pull request merged**